react-components/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "react-components",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
```

Comment:
It's exist as separate script. There is no information about lint in build script in technical description.

Reply:
_Set up eslint and prettier so that the project would not build if there are errors._ 

currently it allows to build with formatting errors as lint check is not included.

Reply:
In the third paragraph of the technical description it is written that there should not be a build on errors, but in the Cross-code-review process, also in paragraph 3 it is written that you can run it with a separate command, so there is a moot point here.